test/suite/test_bug022.py

```diff
@@ -30,27 +30,37 @@
 #       Testing that we don't allow modifies on top of tombstone updates.
 
 import wiredtiger, wttest
+from wtscenario import make_scenarios
 
 class test_bug022(wttest.WiredTigerTestCase):
     uri = 'file:test_bug022'
     conn_config = 'cache_size=50MB'
     session_config = 'isolation=snapshot'
 
+    key_format_values = [
+        ('string', dict(key_format='S', usestrings=True)),
```

Comment:
string -> string-row? a similar change in all the other tests.

Reply:
Fixed these, thanks. This lot was sitting on a different git branch and so got missed by the previous massedit.